ui/src/frontpage/frontpage.tsx

```diff
@@ -1,72 +1,40 @@
 import * as React from 'react';
```

Comment:
Frontpage-komponentti on nyt oikeasti pelkkä frontpage eikä container muille alisivukomponenteille (koodia siirretty toisiin komponentteihin). Julkaisulokikomponenttien näyttäminen ja routet määritellään erikseen main.tsx:ssä. Mielestäni yksinkertaisempi rakenne, as in 1 route -> 1 page, eikä 1 route -> monta pagea, jotka kuitenkin näyttävät koko selaimen kokoista näkymää.